Cargo.toml

```diff
@@ -1,22 +1,25 @@
 [workspace]
 resolver = "2"
 members = [
-    "crates/bdk",
-    "crates/chain",
-    "crates/file_store",
-    "crates/electrum",
-    "crates/esplora",
-    "crates/bitcoind_rpc",
-    "example-crates/example_cli",
-    "example-crates/example_electrum",
-    "example-crates/example_esplora",
-    "example-crates/example_bitcoind_rpc_polling",
-    "example-crates/wallet_electrum",
-    "example-crates/wallet_esplora_blocking",
-    "example-crates/wallet_esplora_async",
-    "nursery/tmp_plan",
-    "nursery/coin_select"
+  "crates/bdk",
+  "crates/chain",
+  "crates/file_store",
+  "crates/electrum",
+  "crates/esplora",
+  "crates/bitcoind_rpc",
+  "example-crates/example_cli",
+  "example-crates/example_electrum",
+  "example-crates/example_esplora",
+  "example-crates/example_bitcoind_rpc_polling",
+  "example-crates/wallet_electrum",
+  "example-crates/wallet_esplora_blocking",
+  "example-crates/wallet_esplora_async",
+  "nursery/tmp_plan",
+  "nursery/coin_select",
 ]
 
 [workspace.package]
 authors = ["Bitcoin Dev Kit Developers"]
+
+[patch.crates-io]
+electrsd = { git = "https://github.com/realeinherjar/electrsd", branch = "einherjar/optional-download" }
```

Comment:
Once https://github.com/RCasatta/electrsd/pull/72 is done, remove this and adjust version

Reply:
Merged, removed